echoshader/dev_class/echo_curtain.py

```diff
@@ -0,0 +1,71 @@
+"""echo_curtain.py module
+
+to get 3D curtains
+"""
+import panel
+
+panel.extension("vtk")
+
+import numpy
+import pyvista
+
+
+def plot_curtain(MVBS_ds, cmp="jet", ratio=0.001):
+    """Get an echogram and histogram with control widgets
+
+    Parameters
+    ----------
+    MVBS : xr.Dataset

```

Comment:
See previous comments regarding this: https://github.com/OSOceanAcoustics/echoshader/pull/50#discussion_r931272181, https://github.com/OSOceanAcoustics/echoshader/pull/50#discussion_r933390453